### PR TITLE
Support Django debug toolbar 3.0

### PR DESCRIPTION
**Pull request by noamkush**
This fixes compatibility with DDT 3.0:
- Moved script load to `ElasticDebugPanel.scripts`
- Small JavaScript fix for strict mode.

### CODE REVIEW

File: elastic_panel/panel.py [hunk 1]
```diff
@@ -56,6 +57,7 @@ class ElasticDebugPanel(Panel):
 
     name = "Elasticsearch"
     template = "elastic_panel/elastic_panel.html"
+    scripts = [static("elastic_panel/js/elastic_panel.js")]
```
Comment (dimastbk) [inline elastic_panel/panel.py:60]: @noamkush this must be a property, otherwise the tests will fail.

Comment (noamkush) [reply elastic_panel/panel.py:60]: I'm not sure which tests are failing, as this passes tests (and in practice it works), but you're right that the base uses a `@property` so I'll change it.

**Comment by Benoss** [comment]
I haven't worked on this for a very long time, but I will try to look at this over the weekend and if I can upgrade the repo to use Github Actions instead of Travis and release a version 3

**Comment by barseghyanartur** [comment]
For me the buttons (`Show Json Body` and `Show Json Response `) don't work.

**settings/dev.py**

```python
DEBUG_TOOLBAR_PANELS = (
    # Defaults
    # 'debug_toolbar.panels.versions.VersionsPanel',
    "debug_toolbar.panels.timer.TimerPanel",
    "debug_toolbar.panels.settings.SettingsPanel",
    "debug_toolbar.panels.headers.HeadersPanel",
    "debug_toolbar.panels.request.RequestPanel",
    "debug_toolbar.panels.sql.SQLPanel",
    # 'debug_toolbar.panels.staticfiles.StaticFilesPanel',
    # 'debug_toolbar.panels.templates.TemplatesPanel',
    "debug_toolbar.panels.cache.CachePanel",
    "debug_toolbar.panels.signals.SignalsPanel",
    "debug_toolbar.panels.logging.LoggingPanel",
    "debug_toolbar.panels.redirects.RedirectsPanel",
    # Additional
    "elastic_panel.panel.ElasticDebugPanel",
)

INSTALLED_APPS += [
    "debug_toolbar",
    "elastic_panel",
]
```

Do they work for you?

**Comment by noamkush** [comment]
If you install my version (this PR) they will work. Try the following:
```
pip install https://github.com/noamkush/django-elasticsearch-debug-toolbar/archive/refs/heads/ddt3_fixes.zip
```

**Comment by barseghyanartur** [comment]
Great, thanks! 
Hey, as a proposal (what I would do in such cases) why don't you re-brand your branch as `django-elasticsearch-debug-toolbar-reloaded` and make a separate PyPI release, maintain the project, etc? And if one day this one would resurrect, the PR is there. :)

**Comment by barseghyanartur** [comment]
OK, if you still have doubts, I can do it based on your branch, make a PyPI release, etc. And add you as a maintainer on both PyPI and GitHub.

**Comment by noamkush** [comment]
@barseghyanartur Sure, I certainly don't have the spare time to take care of it at the moment.

**Comment by barseghyanartur** [comment]
@Benoss:

Wow! Thanks for merging. Could you make a PyPI release from the current master?

**Comment by Benoss** [comment]
Yes I am quickly switching to github actions as Travis.org does not work anymore, adding a few cleanup things like flake8 and black and then trying to release a version 3.0.0

**Pull request merged**